### PR TITLE
feat: bring multi hop wip ui back from the dead

### PR DESCRIPTION
**Pull request by woodenfurniture**
## Description

Resurrects stale WIP ui for multi hop trades so we can restart development. Behind a feature flag defaulting to off.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

NA

## Risk

near-zero risk.

## Testing

Check existing trade confirm UI appears as normal without displaying the broken incomplete multi-hop UI.

### CODE REVIEW

**Comment by woodenfurniture** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5567** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5567" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5567?utm_source=stack-comment).

File: src/state/slices/tradeQuoteSlice/selectors.ts [hunk 1]
```diff
@@ -138,6 +138,9 @@ export const selectTotalProtocolFeeByAsset: Selector<
   quote ? getTotalProtocolFeeByAsset(quote) : undefined,
 )
 
+export const selectIsActiveQuoteMultiHop: Selector<ReduxState, boolean | undefined> =
+  createSelector(selectActiveQuote, quote => (quote ? quote?.steps.length > 1 : undefined))
```
Comment (gomesalexandre) [inline src/state/slices/tradeQuoteSlice/selectors.ts:142]: Do we want to consistently return a boolean here or is undefined intended to signify intent of the quote being possibly nullish?

**Review by gomesalexandre — APPROVED** [review]
Tested locally, confirmed trade/confirm still looks sane without multi-hop swapper UI bits regressing the current one:

Screenshot 2023-11-02 at 11 04 35
Screenshot 2023-11-02 at 11 04 27
Screenshot 2023-11-02 at 11 04 21

File: src/state/slices/tradeQuoteSlice/selectors.ts (hunk 1 above)
Comment (woodenfurniture) [reply src/state/slices/tradeQuoteSlice/selectors.ts:142]: undefined means there is no active quote, meaning `isActiveQuoteMultiHop` is not defined 👍 useful for handling indeterminate state where we dont want app to behave as though there is 1 or 2 hops, but instead there is no quote at all

**Pull request merged**